ci/pull-request-comments.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  bumpDatadogCiComment: `This PR was automatically created because a new version of datadog-ci was published.
+Once merged, please use [this manual workflow](../actions/workflows/release-version.yml) to release the CI integration.`,
+  releaseVersionComment: `Once merged, this PR will automatically create a GitHub release for you.
+The description of the release will exactly match this PR's description. Feel free to edit it.`,
+}
```

Comment:
I'm wondering why is the purpose of this file? Why not inline these messages within the workflow steps?

Reply:
Since those messages are constants, long, multiline, and don't need any Github workflow expression, I thought it would be better to put them in a separate file.

We may update them in the future to add more instructions, so they could become bigger.